src/D2L.CodeStyle.TestAnalyzers/D2L.CodeStyle.TestAnalyzers.csproj

```diff
@@ -14,10 +14,12 @@
     <Copyright>Copyright © D2L Corporation 2018</Copyright>
     <DevelopmentDependency>true</DevelopmentDependency>
 
+    <AnalysisLevel>latest-Recommended</AnalysisLevel>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <IncludeBuildOutput>False</IncludeBuildOutput>
     <IncludeSymbols>True</IncludeSymbols>
-    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <SuppressDependenciesWhenPacking>True</SuppressDependenciesWhenPacking>

```

Comment:
To fix NU5128 

https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5128